website/docs/docs/build/data-tests.md

```diff
@@ -66,7 +66,9 @@ having total_amount < 0
 
 </File>
 
-The name of this test is the name of the file: `assert_total_payment_amount_is_positive`.
+The name of this test is the name of the file: `assert_total_payment_amount_is_positive`. 
+
+Note, that you don't need to include semicolons (;) at the end of the SQL statement in your singular test files as it can cause your test to fail.
```

Comment:
```suggestion
Note, you won't need to include semicolons (;) at the end of the SQL statement in your singular test files as it can cause your test to fail.
```